### PR TITLE
Firefox 11.0 for iOS String Export (With Xcode 8.3.3)

### PR DESCRIPTION
**Pull request by st3fan**
Here is the first string export for Firefox 11.0 for iOS.

### CODE REVIEW

File: nl/firefox-ios.xliff [hunk 1]
```diff
@@ -1009,6 +1003,10 @@
         <target>Wilt u het wachtwoord op %@ bijwerken?</target>
         <note>Prompt for updating a password with on username. The parameter is the hostname of the site.</note>
       </trans-unit>
+      <trans-unit id="Don't Enable">
```
Comment (st3fan) [inline nl/firefox-ios.xliff]: @st3fan Ask Jean / Vijay if these strings actually need to be in here. My guess is not.

File: nl/firefox-ios.xliff [hunk 2]
```diff
@@ -1044,6 +1042,10 @@
         <target>Downloads worden nog niet ondersteund in Firefox.</target>
         <note>The message displayed to a user when they try and perform the download of an asset that Firefox cannot currently handle.</note>
       </trans-unit>
+      <trans-unit id="Enable Push">
```
Comment (st3fan) [inline nl/firefox-ios.xliff]: @st3fan Ask Jean / Vijay if these strings actually need to be in here. My guess is not.

File: nl/firefox-ios.xliff [hunk 3]
```diff
@@ -1129,21 +1134,28 @@
...
+      </trans-unit>
       <trans-unit id="FxA.FirefoxAccount">
         <source>Firefox Account</source>
         <target>Firefox-account</target>
         <note>Settings section title for Firefox Account</note>
       </trans-unit>
       <trans-unit id="FxA.ManageAccount">
-        <source>Manage Account</source>
-        <target>Account beheren</target>
+        <source>Manage Account &amp; Devices</source>
```
Comment (st3fan) [inline nl/firefox-ios.xliff]: @Delphine will this change show up in the web tools?

File: nl/firefox-ios.xliff [hunk 4]
```diff
@@ -1094,10 +1096,13 @@
         <target>Facebook</target>
         <note>Tile title for Facebook</note>
       </trans-unit>
-      <trans-unit id="Firefox will stop syncing with your account, but won’t delete any of your browsing data on this device.">
```
Comment (st3fan) [inline nl/firefox-ios.xliff]: @st3fan What is the bug relating to this change?

File: nl/firefox-ios.xliff [hunk 5]
```diff
@@ -907,11 +906,6 @@
         <target>Nieuw tabblad geopend</target>
         <note>The label text in the Button Toast for switching to a fresh New Tab.</note>
       </trans-unit>
-      <trans-unit id="Copy Address">
```
Comment (st3fan) [inline nl/firefox-ios.xliff]: @st3fan Why was this removed? What is the bug for this one?

Comment (farhanpatel) [reply nl/firefox-ios.xliff]: One is used by the accessibility actions and another by the longpress menu

File: nl/firefox-ios.xliff [hunk 6]
```diff
@@ -3026,9 +3153,8 @@
         <target>Alle tabbladen sluiten</target>
         <note>Label for the button, displayed in the menu, used to close all tabs currently open.</note>
       </trans-unit>
-      <trans-unit id="Menu.CopyURL.Title">
-        <source>Copy URL</source>
-        <target>URL kopiëren</target>
+      <trans-unit id="Menu.CopyAddress.Title">
+        <source>Copy Address</source>
```
Comment (farhanpatel) [inline nl/firefox-ios.xliff]: Copy Address added back in here

File: nl/firefox-ios.xliff (hunk 3 above)
Comment (flodolo) [reply nl/firefox-ios.xliff]: Yes, because there's no target (it's removed by the update script)

File: templates/firefox-ios.xliff [hunk 7]
```diff
@@ -827,6 +823,10 @@
         <source>Do you want to update the password on %@?</source>
         <note>Prompt for updating a password with on username. The parameter is the hostname of the site.</note>
       </trans-unit>
+      <trans-unit id="Don't Enable">
+        <source>Don't Enable</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Use apostrophes, not straight quotes.

Comment (st3fan) [reply templates/firefox-ios.xliff]: I am going to remove these strings.

**Comment by st3fan** [comment]
Uploaded a new PR, closing this one.
